input/intro-notes/StructureDefinition-au-patient-intro.md

```diff
@@ -15,5 +15,38 @@
   - If `Patient.communication.preferred` has not been included, or is set to "false", systems should understand this as the language for the interpreter service is not known.
 - See each Identifier profile page for guidance related to that identifier type.
 
+**Person Recorded Sex Or Gender implementation guidance**
```

Comment:
Remove this heading. Turn "This profile supports blah blah" into a bullet point.

DO NOT have the table as a bullet point. Leave it where it is.

Formatting of guidance in the table is different to the implementation guidance above, e.g. single quotes vs double. Need to apply the AU Base styling.


Does this need to be at the bottom? Can it be under gender identity, and then restart the bullets after the table?

Reply:
Moved guidance to be under the gender identity & applied formatting changes.